src/Command/Issue/IssueShowCommand.php

```diff
@@ -89,20 +84,21 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $styleHelper->section('Body');
         $styleHelper->text(explode("\n", $issue['body']));
 
-        if (true === $input->getOption('with-comments') && count($comments) > 0) {
+        if (true === $input->getOption('with-comments')) {
+            $comments = $tracker->getComments($issueNumber);
             $styleHelper->section('Comments');
             foreach ($comments as $comment) {
-                $styleHelper->listing([
-                    sprintf(
-                        'Comment #%s by %s on %s',
-                        $comment['id'],
-                        $comment['user'],
-                        $comment['created_at']->format('r')
-                    ),
-                    'Link: '.$comment['url'],
-                    '',
-                    wordwrap($comment['body'], 100),
+                $styleHelper->text(sprintf(
+                    '<fg=white>Comment #%s</> by %s on %s',
+                    $comment['id'],
+                    $comment['user'],
+                    empty($comment['created_at']) ? '' : $comment['created_at']->format('r')
```

Comment:
The `created_at` key can be null if there is no date in API data... Maybe it's too much, there is always a date... I'll remove it !